### ISSUE TITLE
Function bind operator fails `no-invalid-this` rule

### ISSUE THREAD

**Issue by vinsonchuong** [babel/eslint-plugin-babel#82]
``` js
function fn() {
  return this.value;
}

const obj = {
  value: 42
};

obj::fn();
```

I can see why `eslint` proper doesn't support this usecase. Can `eslint-plugin-babel` provide a `babel/no-invalid-this` rule to support this usecase?

**Comment by rodrigocnascimento** [comment]
I'm having the same problem. I'm very new to the new JS = P But I want to understand three things about it. Why I can't do it? The "this" is not allowed, even if I use an arrow function? And finally: Why use "babel/no-invalid-this" if I can use this same feature on .eslint.js itself?

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [@babel/eslint-plugin has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
